src/Fixer/PhpUnit/PhpUnitDataProviderStaticFixer.php

```diff
@@ -44,6 +48,18 @@ public function provideSomethingCases() {}
 }
 '
                 ),
+                new CodeSample(
+                    '<?php
+class FooTest extends TestCase {
+    /**
+     * @dataProvider provideSomethingCases
+     */
+    public function testSomething($expected, $actual) {}
+    public function provideSomethingCases() { $this->getData(); }
+}
+',
+                    ['force' => true]
```

Comment:
I would love to have CodeSample of `public function provideSomethingCases() { $this->getData(); }` with `force=>false`, to let the reader see the diff between `force` enabled and disabled.

Reply:
done